### ISSUE TITLE
The Python Support

### ISSUE THREAD

**Issue by allenwoods** [redguardtoo/vscode-matchit#3]
I use evil-matchit in Emacs, thanks for porting it to vscode! Will the Python language support be added as in evil-matchit soon? 

I'm not so familiar with vscode extension but knows bits of elisp. If there is a guide, I would like to help to transport the Python language support from evil-matchit.

**Comment by redguardtoo** [comment]
Yeah, I do have plan to support python. But need refactoring code first.

**Comment by kinredon** [comment]
Any progress here?
